config/crd/patches/webhook_in_tests.yaml

```diff
@@ -14,4 +14,4 @@ spec:
           path: /convert
       conversionReviewVersions:
       - v1
-      - v2
+      - v1beta1
```

Comment:
why did we remove v2?

Reply:
conversionReviewVersion is the version of the Kubernetes resource called ConversionReview which is created by the control plane when it wants to trigger a resource conversion in the operator.
The current supported versions of ConversionReview are v1 and v1beta1